src/vscodeAdapter.ts

```diff
@@ -0,0 +1,62 @@
+import * as vscode from "vscode";
```

Comment:
It feels like this test infrastructure stuff should be in a separate folder rather than at the top-level `src`. Should it be somewhere in the `test` folder? Or, does this need to be included with the extension?

Reply:
fixed.